### ISSUE TITLE
Package has been ignored because it contains invalid configuration.

### ISSUE THREAD

**Issue by stelmakhivan** [dotansimha/graphql-eslint#840]
After updating to latest versions I've been getting the following warning on app startup:

```
warn Package @graphql-codegen/cli has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/cli/package.json.js'
warn Package @graphql-codegen/fragment-matcher has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/fragment-matcher/package.json.js'
warn Package @graphql-codegen/introspection has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/introspection/package.json.js'
warn Package @graphql-codegen/typescript has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/typescript/package.json.js'
warn Package @graphql-codegen/typescript-document-nodes has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/typescript-document-nodes/package.json.js'
warn Package @graphql-codegen/typescript-graphql-files-modules has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/typescript-graphql-files-modules/package.json.js'
warn Package @graphql-codegen/typescript-graphql-request has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/typescript-graphql-request/package.json.js'
warn Package @graphql-codegen/typescript-operations has been ignored because it contains invalid configuration. Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/typescript-operations/package.json.js'
```
I haven't changed anything related to GraphQL Codegen. I'm on TypeScript 4.4.2, and here is the relevant part of my _DEV_ dependencies:

```
"devDependencies": {
	...
    "@graphql-codegen/cli": "2.1.1",
    "@graphql-codegen/fragment-matcher": "3.1.0",
    "@graphql-codegen/introspection": "2.1.0",
    "@graphql-codegen/typescript": "2.1.2",
    "@graphql-codegen/typescript-document-nodes": "2.1.2",
    "@graphql-codegen/typescript-graphql-files-modules": "2.1.0",
    "@graphql-codegen/typescript-graphql-request": "^4.1.2",
    "@graphql-codegen/typescript-operations": "2.1.2",
	...
  }
```
**To Reproduce**
Have the latest versions of GraphQL codegen for TypeScript in your dev dependencies, run your app, check console.

**Expected behavior**
The warning shouldn't display.

**Environment:**

OS: macOS 11.3.1
`@graphql-codegen/...`: See above
NodeJS: 14.17.2
**Additional context**
This wasn't the case before updating to latest versions. Note that this is a minor annoyance, not a breaking bug or anything. Everything still _work as expected_ otherwise.

**Comment by stelmakhivan** [comment]
@dotansimha, I have just installed the latest version of the @graphql-eslint package and I don't see `"./package.json": "./package.json"` in `node_modules/@graphql-eslint/eslint-plugin/package.json`

<img width="1013" alt="Screenshot 2021-12-14 at 21 37 25" src="https://user-images.githubusercontent.com/28428836/146068115-e765c5c5-4350-4640-8b7a-4b640305d0c5.png">

**Comment by dotansimha** [comment]
Thanks @stelmakhivan 
@kamilkisiela @ardatan i think it's an issue with `bob` maybe?

**Comment by kamilkisiela** [comment]
```
Reason: Cannot find module '/path/to/my/app/node_modules/@graphql-codegen/fragment-matcher/package.json.js'
```

`.json.js`, probably `require`. I don't think it's related to bob

**Comment by stelmakhivan** [comment]
> 

When I add `"./package.json": "./package.json"` manually (see this comment https://github.com/dotansimha/graphql-code-generator/issues/6599#issuecomment-974742983) then warnings are fixed

I am using react native in my project.

But after I install the latest version of @graphql-eslint, `"./package.json": "./package.json"` is missed in exports even if I have this line already in code (see https://github.com/dotansimha/graphql-eslint/blob/master/packages/plugin/package.json#L11)

It works perfectly with other libraries, like these:
[react-hook-form](https://github.com/react-hook-form/react-hook-form/pull/3258/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R18)
[zora](https://github.com/lorenzofox3/zora/pull/74/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R20)
[effector](https://github.com/effector/effector/pull/568/files#diff-e0abac90b4f390eafdcf9cac6e8fb2660bb73a04482fa8ecde00c877b8bdd091R8)
[react-final-table](https://github.com/Buuntu/react-final-table/pull/35/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R22)

**Comment by kamilkisiela** [comment]
lol, didn't know you need to put `package.json` in `package.json` to import/require `package.json` :D

**Comment by kamilkisiela** [comment]
so yeah, it gets lost at some point

**Comment by stelmakhivan** [comment]
> lol, didn't know you need to put `package.json` in `package.json` to import/require `package.json` :D

react-native-community/cli#1168

**Comment by B2o5T** [comment]
Fixed in 3.3.0 version
